tests/test_job_management.py

```diff
@@ -1955,6 +1953,106 @@ def x(arg):
     assert len(my_globals['scwl_dict']['scw_list']) > max_length
 
 
+@pytest.mark.parametrize('time_combinations', [[57818.560277777775, 57818.64753472222],
+                                               ['2017-03-06T13:26:48.000', '2017-03-06T15:32:27.000'],
+                                               ['2017-03-06T13:26:48.000', 57818.64753472222],
+                                               [57818.560277777775, '2017-03-06T15:32:27.000']])
+@pytest.mark.parametrize('time_format', ['isot', 'mjd'])
+def test_email_t1_t2(dispatcher_live_fixture,
```

Comment:
make this long-living fixture? To test faster. I doubt multi-threading counts here.